### PR TITLE
Add TotalValue per profile

### PR DESCRIPTION
**Pull request by cyriltovena**
This will allow to compute value per series without having to look at every sample.

I decided to go with v2 meta since we haven't released it yet so it will break in dev. Turns out the repeatedPageIterator also works for non-repeated column :).

### CODE REVIEW

**Review by kolesnikovae — APPROVED** [review]
LGTM

File: pkg/phlaredb/sample_merge.go [hunk 1]
```diff
@@ -365,7 +365,11 @@ func (b *singleBlockQuerier) MergeByLabels(ctx context.Context, rows iter.Iterat
 	defer sp.Finish()
 
 	m := make(seriesByLabels)
-	if err := mergeByLabels(ctx, b.profiles.file, rows, m, by...); err != nil {
+	columnName := "TotalValue"
+	if b.meta.Version == 1 {
+		columnName = "Samples.list.element.Value"
+	}
+	if err := mergeByLabels(ctx, b.profiles.file, columnName, rows, m, by...); err != nil {
```
[the remark this answers is not in the document]
Comment (cyriltovena) [reply pkg/phlaredb/sample_merge.go:372]: Correct.

[the remark this answers is not in the document]
Comment (cyriltovena) [reply pkg/phlaredb/sample_merge.go:372]: v2 is only in dev

**Review by simonswine — APPROVED** [review]
LGTM

**Pull request merged**